Elias/exercise-22.js

```diff
@@ -0,0 +1,83 @@
+let num1 = 60;
+let num2 = 50;
+let num3 = 40;
+
+if ( num1 >= 50 && num2 >= 50 && num1 <= 99 && num2 <= 99) {
+    console.log(true);
+} else {
+    console.log(false);
+}
+
+if ( num1 >= 50 || num2 >= 50 || num3 >= 50 || num3 <= 99 || num1 <= 99 || num2 <= 99) {
+    console.log(true);
+} else {
+    console.log(false);
+}
+
+let a = 10;
+let b = 34;
+let c = 100;
+
+if ( a > b && a > c) {
+    console.log(`a with ${a} is the biggest number!`);
+} else if ( b > a && b > c) {
+    console.log(`b with ${b} is the biggest number!`);
+} else {
+    console.log(`c with ${c} is the biggest number!`);
+}
+// Still works when changing the numbers
+
+
+let string = "Apple";
+let newString = "Py";
+
+if (newString !== string) {
+    newString += string;
+    console.log(newString);
+} else {
+    console.log(string);
+}
+
+let sum = num1 + num2;
+
+if (sum >= 50 && sum <= 80) {
+    console.log(65);
+} else {
+    console.log(80);
+}
+
+let int1 = 3;
+let int2 = 5;
+let sum1 = int1 + int2;
+let sum2 = int1 - int2;
```

Comment:
The result is correct but its always better to use Math.abs() method to calculate the difference, so if you use abs() method the difference is always 2 but if you subtract int1 from int2 the result is -2